src/sentry/replays/serializers.py

```diff
@@ -0,0 +1,13 @@
+from sentry.api.serializers import Serializer
+from sentry.replays.models import ReplayRecordingSegment
+
+
+class ReplayRecordingSegmentSerializer(Serializer):
```

Comment:
if you register this with the django model via decorator you don't have to explicity pass the serializer in the the endpoint
`@register(ReplayRecordingSegment)`